Makefile

```diff
@@ -1,150 +1,105 @@
+comma := ,
+
 .PHONY: help
-help:
-	@echo "make help              Show this help message"
-	@echo 'make services          Run the services that `make dev` requires'
-	@echo "                       (Postgres, Elasticsearch, etc) in Docker Compose"
-	@echo 'make db                Upgrade the DB schema to the latest version'
-	@echo "make dev               Run the app in the development server"
-	@echo "make devdata           Upsert standard development data into the DB, and set"
-	@echo "                       standard environment variables for a development"
-	@echo "                       environment"
-	@echo "make shell             Launch a Python shell in the dev environment"
-	@echo "make sql               Connect to the dev database with a psql shell"
-	@echo "make lint              Run the code linter(s) and print any warnings"
-	@echo "make format            Correctly format the code"
-	@echo "make checkformatting   Crash if the code isn't correctly formatted"
-	@echo "make test              Run the unit tests"
-	@echo "make backend-tests     Run the backend unit tests"
-	@echo "make frontend-tests    Run the frontend unit tests"
-	@echo "make coverage          Print the unit test coverage report"
-	@echo "make functests         Run the functional tests"
-	@echo "make docs              Build docs website and serve it locally"
-	@echo "make checkdocs         Crash if building the docs website fails"
-	@echo "make sure              Make sure that the formatter, linter, tests, etc all pass"
-	@echo "make docker            Make the app's Docker image"
-	@echo "make run-docker        Run the app's Docker image locally. "
-	@echo "                       This command exists for conveniently testing "
-	@echo "                       the Docker image locally in production mode. "
-	@echo "                       It assumes the services are being run using "
-	@echo "                       docker compose in the 'h_default' network."
+help = help::; @echo $$$$(tput bold)$(strip $(1)):$$$$(tput sgr0) $(strip $(2))
+$(call help,make help,print this help message)
 
 .PHONY: services
+$(call help,make services,start the services that the app needs)
 services: args?=up -d --wait
 services: python
 	@docker compose $(args)
 
 .PHONY: db
+$(call help,make db,initialize the DB and upgrade it to the latest migration)
 db: args?=upgrade head
 db: python
-	@tox -qqe dev --run-command 'sh bin/hypothesis --dev init'
-	@tox -qe dev --run-command 'sh bin/hypothesis --dev migrate $(args)'
-
-.PHONY: dev
-dev: build/manifest.json python
-	@tox -qe dev
-
-.PHONY: devssl
-devssl: export H_GUNICORN_CERTFILE=.tlscert.pem
-devssl: export H_GUNICORN_KEYFILE=.tlskey.pem
-devssl: export APP_URL=https://localhost:5000
-devssl: export WEBSOCKET_URL=wss://localhost:5001/ws
-devssl: build/manifest.json python
-	@tox -qe dev
```

Comment:
Note to self: need to re-add this `make devssl` command. This could just go in `h.mk` or it could be added to the cookiecutter.